ntbs-service/Pages/Notifications/Edit/_SaveButton.cshtml

```diff
@@ -7,11 +7,10 @@
     var isDraft = Model.NotificationStatus == NotificationStatus.Draft;
     var pageHasEditableContent = (bool?) ViewData["PageHasEditableContent"] ?? true;
     var currentPage = (string) ViewData["CurrentPage"];
-    var isEditMDRPage = currentPage == NotificationSubPaths.EditMDRDetails;
-    var isEditTreatmentEventsPage = currentPage == NotificationSubPaths.EditTreatmentEvents;
+    var isLastPage = (bool?) ViewData["IsLastPage"] ?? false;
 
     var buttonName = "";
-    if (isEditMDRPage && isDraft)
+    if (isDraft && isLastPage && pageHasEditableContent)
```

Comment:
Could we replace the following block:
```
    if (isDraft && isLastPage && pageHasEditableContent)
    {
        buttonName = "Save";
    }
    else if (!pageHasEditableContent)
    {
        buttonName = "Continue";
    }
    else
    {
        buttonName = isDraft ? "Save and continue" : "Save";
    }
```
with
```
    if (!pageHasEditableContent)
    {
        buttonName = "Continue";
    }
    else if (isDraft && !isLastPage)
    {
        buttonName = "Save and continue";
    }
    else
    {
        buttonName = "Save";
    }
```
I think that's logically the same? It's certainly more readable, if so.